src/scribe_data/extract_transform/languages/English/nouns/format_nouns.py

```diff
@@ -6,31 +6,20 @@
 """
 
 import collections
-import json
 import os
 import sys
 
+PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
+PATH_TO_SCRIBE_DATA_SRC = f"{PATH_TO_SCRIBE_ORG}Scribe-Data/src"
+sys.path.insert(0, PATH_TO_SCRIBE_DATA_SRC)
+from scribe_data.utils import export_formatted_data, load_queried_data
+
 LANGUAGE = "English"
 QUERIED_DATA_TYPE = "nouns"
-QUERIED_DATA_FILE = f"{QUERIED_DATA_TYPE}_queried.json"
-PATH_TO_SCRIBE_ORG = os.path.dirname(sys.path[0]).split("Scribe-Data")[0]
-LANGUAGES_DIR_PATH = (
-    f"{PATH_TO_SCRIBE_ORG}/Scribe-Data/src/scribe_data/extract_transform/languages"
-)
 
 file_path = sys.argv[0]
 
-update_data_in_use = False  # check if update_data.py is being used
-if f"languages/{LANGUAGE}/{QUERIED_DATA_TYPE}/" not in file_path:
-    data_path = QUERIED_DATA_FILE
-else:
-    update_data_in_use = True
-    data_path = (
-        f"{LANGUAGES_DIR_PATH}/{LANGUAGE}/{QUERIED_DATA_TYPE}/{QUERIED_DATA_FILE}"
-    )
-
-with open(data_path, encoding="utf-8") as f:
-    nouns_list = json.load(f)
+nouns_list, update_data_in_use, data_path = load_queried_data(LANGUAGE, QUERIED_DATA_TYPE, file_path)
```

Comment:
Can you add in the arguments explicitly for `load_queried_data` and `export_formatted_data`. Will make it a bit easier for people to follow what's going on when they see what `LANGUAGE`, etc are being passed to.

Reply:
done